### PR TITLE
Docs: Clarify organizing pages into subfolders

### PR DESCRIPTION
**Pull request by frenchy13**
This is a 🔦 documentation change.

## Summary
Adding clarification about the structure of pages in sub folders with example

## Context
It does not address any open issues but does provide clarification on resulting structure/naming of page files/folders which I was having issues understanding at first.

### CODE REVIEW

File: docs/_docs/pages.md [hunk 1]
```diff
@@ -20,7 +20,16 @@ and associated URLs might look like:
...
 ```
 
-If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in our project's source will exist in the `_site` folder when your site builds.
+If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in our project's source will exist in the `_site` folder when your site builds. Each individual page will then be generated in a sub folder mathcing its file name and be called index.html
+
+```sh
+.
+|-- about.md          # => http://example.com/about.html
+|-- Documentation     # folder containing pages
+    └── Doc1.md       # => http://example.com/Documentation/Doc1/index.html
```
Comment (ashmaroli) [inline docs/_docs/pages.md]: If `about.md` generates `/about.html`, then `Documentation/Doc1.md` will generate `/Documentation/Doc1.html` unless there is a `permalink` entry in the page's front matter.
Please update the comment and paragraph above.

Comment (frenchy13) [reply docs/_docs/pages.md]: Fixed as requested. spending too long playing with Jekyll now which has the other structure, sorry .

File: docs/_docs/pages.md [hunk 2]
```diff
@@ -20,7 +20,16 @@ and associated URLs might look like:
 └── contact.html  # => http://example.com/contact.html
 ```
 
-If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in our project's source will exist in the `_site` folder when your site builds.
+If you have a lot of pages, you can organize them into subfolders. Unless there is a permalink entry in the page's front matter the same subfolders that are used to group your pages in your project's source will exist in the `_site` folder when your site builds. Each individual page is generated in said folders as file_name.html
```
Comment (ashmaroli) [inline docs/_docs/pages.md]:
```suggestion
If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in your project's source will then exist in the `_site` folder when your site builds. However, when a page has a *different* permalink set in the front matter, the subfolder at `_site` changes accordingly.
```

File: docs/_docs/pages.md [hunk 3]
```diff
@@ -20,7 +20,16 @@ and associated URLs might look like:
...
+
+```sh
+.
+|-- about.md          # => http://example.com/about.html
+|-- Documentation     # folder containing pages
+    └── Doc1.md       # => http://example.com/Documentation/doc1.html
```
Comment (ashmaroli) [inline docs/_docs/pages.md]: Does the filename get *downcased* at your end? Or is it just a typo?
On my Windows system, the **file path** is `_site\Documentation\Doc1.html`

Comment (frenchy13) [reply docs/_docs/pages.md]: Just a typo, I will correct it now

File: docs/_docs/pages.md [hunk 4]
```diff
@@ -20,7 +20,16 @@ and associated URLs might look like:
...
 ```
 
-If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in our project's source will exist in the `_site` folder when your site builds.
+If you have a lot of pages, you can organize them into subfolders. The same subfolders that are used to group your pages in your project's source will then exist in the `_site` folder when your site builds. However, when a page has a *different* permalink set in the front matter, the subfolder at `_site` changes accordingly.
+
+```sh
+.
+|-- about.md          # => http://example.com/about.html
+|-- Documentation     # folder containing pages
+    └── Doc1.md       # => http://example.com/Documentation/Doc1.html
```
Comment (DirtyF) [inline docs/_docs/pages.md]: we should not use capitals in a default example, please stick with lowercase.

Comment (frenchy13) [reply docs/_docs/pages.md]: Fixed, thought it may have added clarification. I'm new to github etc so wasn't aware of the standard.

**Comment by DirtyF** [comment]
@jekyllbot: merge +docs

**Pull request merged**